.vscode/settings.json

```
@@ -0,0 +1,13 @@
+{
+    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": [
+        "--extend-ignore=E201,E202,E203",
+        "--max-line-length=120"
+    ],
+    "python.testing.pytestEnabled": true,
+    "python.testing.pytestArgs": [
+        "--rootdir",
+        "${workspaceFolder}/test"
+    ],
```

```suggestion
    "python.testing.pytestArgs": [
        "--rootdir",
        "${workspaceFolder}/test"
    ],
```
```suggestion
```

I tried your variant out locally but VS Code wasn't able to run individual tests with it. Removing it made it work for all tests and individually selected tests.

---

Strangely, removing this made it stop working for me! I've added a commit that invokes `pytest` with `python -m`. Does that work for you?

---

Unfortunately not, the "Python test log" output says:
```
============================= test session starts =============================
platform win32 -- Python 3.9.5, pytest-7.1.1, pluggy-1.0.0
rootdir: c:\Users\maik\git\covjson-validator\test
collected 0 items

- generated xml file: C:\Users\maik\AppData\Local\Temp\tmp-23440k5B13YprZ3lR.xml -
============================ no tests ran in 0.03s ============================
ERROR: file or directory not found: ./domain_types/test_grid.py::test_extra_multi_coordinate_axis

ERROR conda.cli.main_run:execute(33): Subprocess for 'conda run ['python', 'c:\\Users\\maik\\.vscode\\extensions\\ms-python.python-2022.4.0\\pythonFiles\\get_output_via_markers.py', '-m', 'pytest', '--override-ini', 'junit_family=xunit1', '--junit-xml=C:\\Users\\maik\\AppData\\Local\\Temp\\tmp-23440k5B13YprZ3lR.xml', '--rootdir', 'c:\\Users\\maik\\git\\covjson-validator/test', './domain_types/test_grid.py::test_extra_multi_coordinate_axis']' command failed.  (See above for error)
```

When I remove the rootdir option and the pytest path option it looks like this:
```
============================= test session starts =============================
platform win32 -- Python 3.9.5, pytest-7.1.1, pluggy-1.0.0
rootdir: c:\Users\maik\git\covjson-validator
collected 2 items

test\test_axis.py ..                                                     [100%]

- generated xml file: C:\Users\maik\AppData\Local\Temp\tmp-23440zeOm4F7YeTd4.xml -
============================== 2 passed in 0.11s ==============================
```

---

OK, I can't find a way to make this consistently work for both of us. I suggest that I just remove `settings.json` from the PR and we'll just keep our locally-working versions.